beetsplug/parentwork.py

```diff
@@ -0,0 +1,142 @@
+                            partof=True
+                            while partof:
+                                partof=False
+                                if 'work-relation-list' in work_info['work']:
+                                    for work_father in work_info['work']['work-relation-list']:
+                                        if work_father['type'] == 'parts': 
+                                            if 'direction' in work_father:
+                                                if work_father['direction'] == 'backward':
+                                                    father_id=work_father['work']['id']
+                                                    partof=True
+                                                    work_info=musicbrainzngs.get_work_by_id(father_id, includes=["work-rels", "artist-rels"])
+                            if 'artist-relation-list' in work_info['work']:
+                                for artist in work_info['work']['artist-relation-list']:
+                                    if artist['type']=='composer':
+                                        if artist['artist']['name'] in parent_composer:
+                                            pass
+                                        else: 
+                                            parent_composer.append(artist['artist']['name'])
+                                            parent_composer_sort.append(artist['artist']['sort-name'])
+                            else:
+                                print('no composer')
+                                print('add one at')
+                                print('https://musicbrainz.org/work/'+ work_info['work']['id'])
+                            if work_info['work']['title'] in parent_work:
+                                pass
+                            else: 
+                                parent_work.append(work_info['work']['title'])
+                                if 'disambiguation' in work_info['work']:
+                                    parent_work_disambig.append(work_info['work']['disambiguation'])
+                except musicbrainzngs.musicbrainz.NetworkError:
+                    continue
+                    i=i+1
+                break    
+            
+            item['parent_work']          = parent_work
+            item['parent_work_disambig'] = parent_work_disambig
+            item['work']                 = work
+            item['work_disambig']        = work_disambig
+            item['performer']            = performer
+            item['performer_sort']       = performer_sort
+            item['parent_composer']      = parent_composer
+            item['parent_composer_sort'] = parent_composer_sort
+            
+            item.try_write()
+            item.store(fields=['parent_work','parent_work_disambig','work','work_disambig','performer','performer_sort','parent_composer','parent_composer_sort'])
```

Comment:
You can just use `Item.store()` without the field list. This may be the source of the crash you're seeing.